doc/requirements.txt

```diff
@@ -0,0 +1,61 @@
+.. _requirements:
+
+
+Requirements
+============
+
+Theano is mainly develeopped on 64-bit Linux machines. 
+
+These libraries are common to all Linux installations and should be installed
+prior to using Theano.
+
```

Comment:
After discussion with Pascal and Arnaud, we decided to further modify it:
(I will give you more details in person)

1. Add miniconda as the default way to install prerequisites (will need to repeat the libgpuarray instructions)
2. Install MKL if miniconda and linux. Install openblas else.
3. Move here the GPU installation instructions